src/System.Management.Automation/engine/lang/scriptblock.cs

```diff
@@ -647,16 +647,17 @@ internal ReadOnlyCollection<PSTypeName> OutputType
         /// <remarks>
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-macos`
Language Primitive Tests.Convert ScriptBlock to delegate type
```powershell
Unable to find type [System.Func].
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/engine/Api/LanguagePrimitive.Tests.ps1: line 149
149:         $handlerReturnEnum = [System.Func[string, TestEnum]] { param([string]$str) "Music" }

```